src/schedule.rs

```diff
@@ -556,9 +636,7 @@ mod test {
     #[test]
     fn test_next_after_past_date_next_year() {
         // Schedule after 2021-10-27
-        let starting_point = Utc
-            .with_ymd_and_hms(2021, 10, 27, 0, 0, 0)
-            .unwrap();
+        let starting_point = Utc.with_ymd_and_hms(2021, 10, 27, 0, 0, 0).unwrap();
```

Comment:
Apologies for accidentally committing an automated formatting change. If you would like to see it reverted, let me know!